ccm_web/server/localTypes/ltijs/index.d.ts

```diff
@@ -122,6 +123,13 @@ declare module 'ltijs' {
     lineItem: string
   }
 
+  interface UserInfo {
+    given_name: string
+    family_name: string
+    name: string
+    email: string
+  }
+
   interface IdToken {
```

Comment:
@pushyamig, do you want me to add the `custom` key-value pair here as well?

Reply:
Yeah, Make sense to have it if you are touching this code. Here is how I did it? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/66/files#diff-4be8caff9c88221f677afdb7f6238e2ffb00f1a23a40c1de9720c2a6a0a97e25

Reply:
K, I'll probably get to this tomorrow. Thanks

Reply:
@pushyamig, please take a look at this commit: https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/72/commits/1374c30e01b5ea0de55c2fa6ab8d728ce55364c3

I based these changes on the ltijs docs version of the "idToken" and "ContextToken" here: https://cvmcosta.me/ltijs/#/provider?id=authentication-and-routing I also compared them to what Canvas was sending. I guess I assumed that what was in the published types before (and other WIP) I saw was still up to date.

Reply:
I see that you try to get to bottom of it for better understanding. Nice to see you are thinking that way.